### ISSUE TITLE
Create DNS record fails if record already exists

### ISSUE THREAD

**Issue by fillup** [terraform-providers/terraform-provider-cloudflare#10]
tl;dr; attempting to create a resource that already exists should be treated as success and information loaded into state. 

### Terraform Version
Version: 0.9.8

### Affected Resource(s)
Please list the resources as a list, for example:
- cloudflare_record

### Terraform Configuration Files
```hcl
provider "cloudflare" {
  email = "${var.cloudflare_email}"
  token = "${var.cloudflare_token}"
}

resource "cloudflare_record" "foobar" {
  domain = "example"
  name   = "terraform"
  value  = "192.168.0.11"
  type   = "A"
  ttl    = 3600
  priority = 50
  proxied = false
}
```

### Expected Behavior
If the DNS record already exists and matches what is defined in the TF configuration, attempting to create via Terraform should (I believe) consider it existing as success and import attributes about the record into the state. 

This is also important because the cloudflare_record resource does not support importing existing records. So to move from managing via Cloudflare directly to managing via Terraform requires deleting records in Cloudflare and then recreating via Terraform which could cause downtime. 

### Actual Behavior
Terraform errors that it cannot create the record.

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. Create DNS entry directly in Cloudflare
2. Attempt to create it via Terraform by updating config above and running `terraform apply`

**Comment by stack72** [comment]
Hi @fillup 

thanks for the request here - unfortunately, this is not something we would support. This is why we have designed `import` functionality in Terraform. That way we can import existing infra into Terraform management

Unfortunately, there is already an ongoing issue about import in Cloudflare - https://github.com/terraform-providers/terraform-provider-cloudflare/issues/5

Therefore, I am going to close this out as a duplicate so that any further  help / scenarios for implementation would go into that issue

Hope this sounds ok?

Paul

**Comment by fillup** [comment]
Fair enough, thanks.

**Comment by katlimruiz** [comment]
I still haev this error
